### ISSUE TITLE
Content Tree: Custom Action Bar buttons are not sized correctly

### ISSUE THREAD

**Issue by coliff** [trimble-oss/modus-web-components#2162]
See screenshot (from https://modus-web-components.trimble.com/?path=/docs/components-content-tree--multi-selection) showing active state here:

![image](https://github.com/trimble-oss/modus-web-components/assets/1212885/a7a584c9-57c9-4ccb-99cf-d2d8c763bd6a)

We should specify height and width of the button, or set consistent vertical/horizontal padding to fix this issue.

**Comment by coliff** [comment]
@ElishaSamPeterPrabhu Maybe the modus-button should have `icon-only` on it e.g. `icon-only="add"` ?

**Comment by ElishaSamPeterPrabhu** [comment]
> @ElishaSamPeterPrabhu Maybe the modus-button should have `icon-only` on it e.g. `icon-only="add"` ?

Yes we can do that but the icon map doesn't contain for the other icons 

So where should we update them?

**Comment by coliff** [comment]
...or instead use some custom CSS like:

`height: 28px; width: 28px; padding: 8px; margin-left: 8px`

**Comment by ElishaSamPeterPrabhu** [comment]
> height: 28px; width: 28px; padding: 8px; margin-left: 8px

after applying the css
![image](https://github.com/user-attachments/assets/c4074cfd-dccd-4e22-a813-0bf0b0fe4689)
Is this the expected look?

**Comment by coliff** [comment]
not exactly, I just included that CSS as a starting point which improves things somewhat.

**Comment by ElishaSamPeterPrabhu** [comment]
> > Is this the expected look?
> 
> not exactly, I just included that CSS as a starting point which improves things somewhat.

@coliff 
There are two points to address

1. #1466 has a similar need to decide between modus button with icon-only property or the custom css with svg.
2. Is there a reference on how the custom bar should look as its not available in figma?
